man/bubbleplot.Rd

```diff
@@ -16,31 +16,33 @@ bubbleplot(x, \dots)
            rev=FALSE, type="p", ylim=NULL, xlab=NULL, ylab=NULL,
            pch=c(16,1), cex.points=1, col="black", bg=par("bg"), \dots)
 
-\method{bubbleplot}{formula}(formula, data, subset, na.action=NULL,
-           \dots)
+\method{bubbleplot}{formula}(formula, data, subset, na.action=NULL, \dots)
 }
 \arguments{
   \item{x}{a vector of values for the horizontal axis. Can also be a
     2-dimensional matrix or table (x values in column names and y values
-    in rows), or a data frame containing \code{x}, \code{y}, and
-    \code{z} in that order.}
+    in row names), or a data frame containing \code{x}, \code{y}, and
+    \code{z} in that order. If the data frame contains column names
+    \code{x}, \code{y}, and \code{z} then they will be used for
+    plotting.}
   \item{\dots}{passed to \code{plot} and \code{points}.}
   \item{y}{a vector of values for the vertical axis.}
   \item{z}{a vector of values determining the bubble sizes.}
-  \item{std}{whether to standardize \code{z} by dividing with
-    \code{mean(z)}.}
-  \item{pow}{a power coefficient for the bubble sizes (\code{0.5} is the
-    square root).}
+  \item{std}{whether to standardize
+    \code{z = abs(z) / mean(abs(z))}.}
+  \item{pow}{a power coefficient for the bubble sizes. The
+    \code{pow = 0.5} (square root) is a good default, where a \code{z}
```

Comment:
The documentation for `std` and `pow` are probably a bit much for the parameter descriptions.  Perhaps the details of the calculation would probably be better put into a `\details` section

Reply:
Great idea. Done: fcc94df